posthog/batch_exports/http.py

```diff
@@ -65,12 +65,13 @@ def validate_date_input(date_input: Any, team: Team | None = None) -> dt.datetim
         # As far as I'm concerned, if you give me something that quacks like an isoformatted str, you are golden.
         # Read more here: https://github.com/python/mypy/issues/2420.
         # Once PostHog is 3.11, try/except is zero cost if nothing is raised: https://bugs.python.org/issue40222.
-        parsed = dt.datetime.fromisoformat(date_input.replace("Z", "+00:00"))
+        parsed = dt.datetime.fromisoformat(date_input)
```

Comment:
Not needed anymore, I think that with py 3.11 the Z suffix became accepted as valid iso format.